hack/olm-registry/olm-artifacts-template.yaml

```diff
@@ -15,7 +15,7 @@ parameters:
 metadata:
   name: selectorsyncset-template
 objects:
-- apiVersion: hive.openshift.io/v1alpha1
```

Comment:
What's the reason for this change?

Reply:
We should be using hive/v1 everywhere by now

Reply:
Has the operator been tested with hive v1?

Reply:
https://github.com/openshift/cloud-ingress-operator/pull/48 this PR has been merged awhile back. It was just never generated. afaik it's OK be be rolled out

Reply:
IMO should be in a different commit/PR since if you revert this PR this will be reverted as well. But I won't block PR on this

Reply:
I agree with Taylor here, I wouldn't 👎 but if we can easily split it out, we should.